File: 7kue/Is every value in the array an array/solve-1.js
const arrCheck = value => {
  for (let i of value) {
    if (!Array.isArray(i)) {
      return false;
    } 
  }
  return true;
}

console.log(arrCheck([[1, 2, 3], [4, 5, 6], 7])) // Output: false
console.log(arrCheck([[1, 2, 3], [4, 5, 6]])) // Output: true

File: 7kue/Is every value in the array an array/solve-1.py
def arr_check(arr):
  for i in arr:
    if type(i) != list:
      return False
  return True


print(arr_check([[1, 2, 3], [4, 5, 6], 7]))  # Output: False
print(arr_check([[1, 2, 3], [4, 5, 6]]))  # Output: True